### ISSUE TITLE
How does editing styles work exactly?

### ISSUE THREAD

**Issue by jiayounokim** [charmbracelet/glow#201]
Quoting you:
"You could edit your own style JSON and tweak it! Check out the documentation and style definitions at https://github.com/charmbracelet/glamour/tree/master/styles"

But how do I make it all work glow and glamour together?

1. I cloned https://github.com/charmbracelet/glow
2. I did `go build`
3. now if I want to change styles how can I exactly do it? please explain in steps.

Do I need to fork this https://github.com/charmbracelet/glamour/ and then make changes? and if yes, then how would glow and glamour communicate with my fork versions?

TLDR: I am confused but still want to edit styles.

**Comment by muesli** [comment]
You can simply put a style json file (as you can find them in the glamour repo) anywhere on your disk and then point glow to it with the --style flag. If you have a finished style you can open a PR for glamour!
